vaadinfrontend/src/main/java/life/qbic/datamanager/views/projects/project/experiments/ExperimentInformationPage.java

```diff
@@ -79,7 +92,8 @@ private void setupBoard(ProjectNavigationBarComponent projectNavigationBarCompon
     mainComponents.add(projectNavigationBarComponent, experimentDetailsComponent);
 
     rootRow.add(mainComponents, 3);
-    rootRow.add(experimentListComponent, 1);
+    itemCollection = ExperimentItemCollection.create();
```

Comment:
While it makes sense that the itemCollection is not styled as a `PageArea` 
it'd be a good to introduce a dedicated "ExperimentSupportArea" which follows the PageArea style and handles the logic of the itemCollection.

Reply:
For me it is not clear, how the UI is supposed to look like, when more than one support component is added in the support section. Thus I left it simple.